recipes/newrelic/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+if [ "$(uname)" == "Darwin" ];
+then
+    export CC=clang
+    export CXX=clang++
+
+    export MACOSX_VERSION_MIN=10.7
+    export CXXFLAGS="-mmacosx-version-min=${MACOSX_VERSION_MIN}"
+    export CXXFLAGS="${CXXFLAGS} -stdlib=libc++ -std=c++11"
+    export LINKFLAGS="-mmacosx-version-min=${MACOSX_VERSION_MIN}"
+    export LINKFLAGS="${LINKFLAGS} -stdlib=libc++ -std=c++11 -L${LIBRARY_PATH}"
+fi
+
+"${PYTHON}" setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
Do you need all that on OS X? If not it would be nice to remove the build scripts and add a single `script` line in the `meta.yaml` build section.